src/components/MyDocument/index.js

```diff
@@ -206,7 +209,7 @@ const MyDocument = ({ bedLayout, title, colsPerPage, data, census }) => {
       <Page size="letter" style={pdfStyles.page}>
         <View style={pdfStyles.header}>
           <View style={pdfStyles.titleRoot}>
-            <Text>{title}</Text>
+            <Text>{title || " "}</Text>
```

Comment:
This is the current solution to fixing this error. Apparently, the <Text> and/or <View> components don't like to be empty. It seems to mess with their style calculations if align or height can't be figured out. 

Need to consider a cleaner way of doing this at the top, prior to the return